### PR TITLE
Hello world initial commit

### PR DESCRIPTION
**Pull request by dgonzalez**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
```
Comment (dgonzalez) [inline main.py:4]: This is not going to work.

**Review by CristianMenguer — APPROVED** [review]
well done!

**Comment by douglasfunkler** [comment]
Test

**Review by Altemir-Ap — COMMENTED** [review]
It's gonna work xD

**Review by jucielyrodrigues — COMMENTED** [review]
all good

**Pull request merged**